readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -389,8 +178,3 @@ def yaml_load_safely(content):
     information loss.
     """
     return yaml.load(content, Loader=SafeLoader)
```

Comment:
I think we can remove all code related to `SafeLoader` and just call safe_load now.

Reply:
We can't yet because it's used to get the final doctype for the version at https://github.com/readthedocs/readthedocs.org/blob/8440aab6a0be1db5e8425d5bb96dee2802e164b8/readthedocs/doc_builder/backends/mkdocs.py#L66-L68

This code will be eventually removed since the doctype doesn't have too much sense anymore now that we support many doctools more. We should probably need to have a conversation about how to delete doctypes completely in a safe way.

Reply:
yeah, but don't think you need the custom loader to just read that setting.

Reply:
No if you call `safe_load()`

Reply:
The custom loader was added to don't set those values to `None` after reading them (we were using `safe_load` previously).

Reply:
Cool. Good point! I will delete this chunk of code then 👍🏼